authui/src/authflowv2/document.css

```diff
@@ -0,0 +1,13 @@
+@layer component {
+  :root {
+    --document-pointer-event: auto;
+  }
+
+  html[data-inline-preview="true"] {
+    --document-pointer-event: none;
+  }
+
+  html {
+    pointer-events: var(--document-pointer-event);
+  }
```

Comment:
Should this be logic of the iframe? i.e. Set `pointer-events: none` in iframe instead of authui?

Reply:
But pressing the buttons in inline preview mode could cause errors. Lets keep this.

Reply:
I think i will apply to authui coz even not in iframe, it should not be interactable